management/mailconfig.py

```diff
@@ -125,13 +139,42 @@ def get_mail_users_ex(env, with_archived=False):
 	users = []
 	active_accounts = set()
 	c = open_database(env)
-	c.execute('SELECT email, privileges FROM users')
-	for email, privileges in c.fetchall():
+	c.execute('SELECT email, privileges, quota FROM users')
+	for email, privileges, quota in c.fetchall():
 		active_accounts.add(email)
 
+		(user, domain) = email.split('@')
+		box_size = 0
+		box_quota = 0
+		percent = ''
+		try:
+			dirsize_file = os.path.join(env['STORAGE_ROOT'], 'mail/mailboxes/%s/%s/maildirsize' % (domain, user))
```

Comment:
Instead of reading the "maildirsize" file directly, which is specific to the maildir quota backend, why not execute ``doveadm -f json quota get -u <email>`` and parse the json, which contains both quota ("limit") in KiB, current size ("value") also in KiB, and percent used ("percent"). The "STORAGE" type is what you want. "MESSAGE" is a count of messages. The json looks like this:
```doveadm -f json quota get -u "test@abc.com"
[{"root":"User quota","type":"STORAGE","value":"2","limit":"5120","percent":"0"},{"root":"User quota","type":"MESSAGE","value":"1","limit":"-","percent":"0"}]```